CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
 ## Unreleased
+ - Map can be used without an active navigation, navigation can be started anytime with `startNavigation(route: Route)`.
```

Comment:
Did you intend to have two changelog entries? This one seems redundant with the one you added at the end. Can you consolidate them?